### ISSUE TITLE
loss

### ISSUE THREAD

**Issue by Whitecatwarrior** [tegg89/SRCNN-Tensorflow#59]
hello，when I train the model, the loss of the printout becomes strange and I don'tknow if this is normal . Is there anyone can tell me about it?
`
Epoch: [11882], step: [926720], time: [92697.6400], loss: [0.00161069]

Epoch: [11882], step: [926730], time: [92698.6210], loss: [0.00290118]

Epoch: [11882], step: [926740], time: [92699.6220], loss: [0.00086975]

Epoch: [11882], step: [926750], time: [92700.6011], loss: [0.00169979]

Epoch: [11882], step: [926760], time: [92701.6095], loss: [0.00303134]

Epoch: [11882], step: [926770], time: [92702.6365], loss: [0.00131717]

Epoch: [11882], step: [926780], time: [92703.6965], loss: [0.00106838]
`

**Comment by Whitecatwarrior** [comment]
when I train the model through gpu the loss become nan.....
